modules/explorer/explorer.go

```diff
@@ -77,7 +77,7 @@ func New(cs modules.ConsensusSet, persistDir string) (*Explorer, error) {
 		return nil, err
 	}
 
-	err = cs.ConsensusSetSubscribe(e, recentChange)
+	err = cs.ConsensusSetSubscribe(e, recentChange, nil)
```

Comment:
Not sure about this one. Maybe we could pass the `sigChan` directly? Doesn't feel like a clean solution though.